clients/src/main/java/org/apache/kafka/common/requests/UpdateMetadataRequest.java

```diff
@@ -123,14 +129,14 @@ public UpdateMetadataRequest(int controllerId, int controllerEpoch, Set<BrokerEn
     }
 
     /**
-     * Constructor for version 1.
+     * Constructor for version 2.
```

Comment:
Note that this constructor is only used in tests. Does it even make sense to keep it?